cdap-elastic/src/test/java/io/cdap/cdap/metadata/elastic/ElasticsearchMetadataStorageTest.java

```diff
@@ -503,6 +515,106 @@ public void testNumericQueries() throws IOException {
     cleanMetadataStorage(records);
   }
 
+  @Test
+  public void testDateParser() throws IOException {
+    String dateString = "2019-08-05";
+    Assert.assertEquals(new Long(1564988400000L), QueryParser.parseDate(dateString));
+  }
+
+  @Test
+  public void testParseDateQuery() throws IOException {
```

Comment:
this should also be part of `QueryParserTest`. Also remove `IOException` since its not being thrown by the test

Reply:
Ok